### PR TITLE
1 user can pick a day for date

### PR DESCRIPTION
**Pull request by mduguil**
Animated GIF-downsized_large (1)
Animated GIF-downsized_large
drop

### CODE REVIEW

File: client/app.jsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import Home from './pages/home';
+// import Home from './pages/home';
+import DateForm from '../client/components/dateForm';
 
 export default class App extends React.Component {
   render() {
-    return <Home />;
+    // return <Home />;
```
Comment (thebearingedge) [inline client/app.jsx]: Please do not include commented out code in pull requests.

File: client/components/dateForm.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    super(props);
+    const selectedMonth = new Date().getMonth();
+    const selectedYear = new Date().getFullYear();
+    const dayNames = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    this.state = {
+      selectedMonth,
+      selectedYear,
+      dayNames,
+      day: new Date().getDate(),
+      monthNames: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
```
Comment (thebearingedge) [inline client/components/dateForm.jsx]: The names of the month don't change, so you shouldn't put them in state.

File: server/public/styles.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
+* {
+  font-family: Roboto, sans-serif;
+}
+
+.form-title {
+  margin-top: 43px;
+  font-size: 2.3rem;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: If everything is going to be 43 px from the top, just add a padding top of 43px to the body element so you don't have to put this margin on every single page title

File: server/public/styles.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+.form-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+form {
+  width: 340px;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: What about on Desktop? Your figma for desktop uses more space than that. Add a container class instead, like in your css responsive layout exercise.

File: server/public/styles.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  font-size: 1.3rem;
+  color: #8d8d8d;
+  font-weight: bold;
+}
+
+.day-options {
+  display: inline-block;
+  position: relative;
+  left: 50px;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: Please don't use position relative to move things around. instead divide up the boxes into columns (like in your css grid classes exercise)

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please take your time to structure your code and match your figma

**Comment by mduguil** [comment]
aa

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**